staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -127,8 +127,8 @@ var (
 		},
 		[]string{priorityLevel},
 	)
-	// PriorityLevelConcurrencyPairVec creates pairs that observe concurrency for priority levels
-	PriorityLevelConcurrencyPairVec = NewSampleAndWaterMarkHistogramsPairVec(clock.RealClock{}, time.Millisecond,
+	// PriorityLevelConcurrencyVec creates gauges of concurrency broken down by phase, priority level
+	PriorityLevelConcurrencyVec = NewSampleAndWaterMarkHistogramsVec(clock.RealClock{}, time.Millisecond,
```

Comment:
For consistency with other names, maybe call it:

PriorityLevelConcurrencyGaugeVec

?

Reply:
Good catch, thanks.
Done in the force-push to ad9cab90551.